source/class/qx/tool/compiler/resources/Manager.js

```diff
@@ -293,23 +293,15 @@ qx.Class.define("qx.tool.compiler.resources.Manager", {
      * @return {Asset?} the asset, if found
      */
     getAsset(srcPath, create) {
-      let pos = srcPath.indexOf(":");
-      let library = null;
-      if (pos > -1) {
-        let ns = srcPath.substring(0, pos);
-        library = this.__analyser.findLibrary(ns);
-        srcPath = srcPath.substring(pos + 1);
-      } else {
-        library = this.findLibraryForResource(srcPath);
-      }
-      
+      let library = this.findLibraryForResource(srcPath);
       if (!library) {
         this.warn("Cannot find library for " + srcPath);
         return null;
       }
 
+
       let resourceDir = path.join(library.getRootDir(), library.getResourcePath());
-      srcPath = path.relative(resourceDir, path.join(resourceDir, srcPath));
+      srcPath = path.relative(resourceDir, path.isAbsolute(srcPath)?srcPath:path.join(resourceDir, srcPath));
```

Comment:
this will cause problems - just about everywhere, the asset's path is expected to be relative to the resource directory of the library; `asset.getSourceFilename()` will join the paths, and uses it to check whether the resource is in the theme or resource directory.

If srcPath is absolute it should be reduced to relative

Reply:
The srcPath is relative to resourceDir. Problem occours if srcPath is absolute - then the join with resourceDir will add resourceDir to the absolute path

Reply:
oh, sorry you're right